src/datadog/w3c_propagation.cpp

```diff
@@ -300,6 +308,7 @@ Expected<ExtractedData> extract_w3c(
     return result;
   }
 
+  result.datadog_w3c_parent_id = "0000000000000000";
```

Comment:
Curious to know: Is there a reason to use 0s id instead of an empty field somehow? Is the all 0s value a special value somewhere else?

Reply:
It's from the RFC. I think it's an optimization for the backend.